source/ShipInfoDisplay.cpp

```diff
@@ -248,8 +248,8 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 			string bayLabel = bayType;
 			transform(bayLabel.begin(), bayLabel.end(), bayLabel.begin(), ::tolower);
 			
-			attributeLabels.push_back(bayLabel + " bays:");
-			attributeValues.push_back(to_string(ship.BaysFree(bayType)));
+			attributeLabels.emplace_back(bayLabel + " bays:");
+			attributeValues.emplace_back(to_string(ship.BaysFree(bayType)));
```

Comment:
You should not use `BaysFree` here

Reply:
Agreed, I used the value cached earlier in this function.